code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -10,7 +10,7 @@
  */
 /obj/item/clothing/suit/bluetag
 	name = "blue laser tag armour"
-	desc = "Blue Pride, Galaxy Wide."
+	desc = "Blue Pride, World Wide."
```

Comment:
Issue still stands, just make it "Worldwide" its that simple

Reply:
i didnt even touch the redtag

Reply:
Yes because that's a suggested change